### PR TITLE
Docs 19.03.07 release (Week 46, 2024)

### PR DESCRIPTION
**Pull request by Nadia-JSch**
Jira issues: 584, 585, 586, 587

### CODE REVIEW

File: docs/administration/appliance-manager/enable-or-disable-tls-versions-from-appliance-manager.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ sidebar_position: 8
 
 Device42 allows you to restrict TLS versions for Appliance Manager Access. The radio buttons shown below control this feature.
 
-**Note:** As of v16.23.00, all users will be upgraded to **TLS v1.2 by default** for the Appliance Manager and Device42. You can still change your setting to _Enable TLS v.1.1 and v1.2_ or _Enable All TLS_.
+**Note:** All users will be upgraded to **TLS v1.2 by default** for the Appliance Manager and Device42. You can still change your setting to _Enable TLS v.1.1 and v1.2_ or _Enable All TLS_.
```
Comment (zaneD42) [inline docs/administration/appliance-manager/enable-or-disable-tls-versions-from-appliance-manager.md:10]: The appliance manager now defaults to 'Enable TLS v1.2 only'. You an change the TLS settings in the appliance manager to _Enable TLS v.1.1 and v1.2_ or _Enable All TLS_.

File: docs/getstarted/installation/windows-service-installation.mdx [hunk 2]
```diff
@@ -6,7 +6,7 @@ sidebar_position: 8.5
 import ThemedImage from '@theme/ThemedImage';
 import useBaseUrl from '@docusaurus/useBaseUrl'
 
-The Device42 Windows Service Wrapper is an executable (available separately for i386 32-bit and 64-bit architectures) that can be installed as a service. Download the Windows Service Wrapper from [the Device42 Autodiscovery page](https://www.device42.com/autodiscovery/). The service wrapper calls the Device42 Windows agent that you can download from your Device42 Main Appliance (MA).
+The Device42 Windows Service Wrapper is an executable (available separately for i386 32-bit and 64-bit architectures) that can be installed as a service. Download the Windows Service Wrapper from [the Device42 Autodiscovery page](https://www.device42.com/autodiscovery/). The service wrapper calls the Device42 Windows agent, whichyou can download from your Device42 Main Appliance (MA).
```
Comment (zaneD42) [inline docs/getstarted/installation/windows-service-installation.mdx:9]: which you

File: docs/getstarted/discovery-hub.mdx [hunk 3]
```diff
@@ -191,7 +191,7 @@ Click on the **+ Add schedule button** to schedule the scan.
 
 #### Additional Details
 
-This information is accurate as of release v18.12.00, with more usability enhancements on their way.
+Device42 is continuously works to add new usability enhancements.
```
Comment (zaneD42) [inline docs/getstarted/discovery-hub.mdx]: This provides little value, remove sentence and ensure the rest of the page makes sense.

Comment (Nadia-JSch) [reply docs/getstarted/discovery-hub.mdx]: Removed

File: docs/getstarted/installation/windows-service-installation.mdx (hunk 2 above)
Comment (Nadia-JSch) [reply docs/getstarted/installation/windows-service-installation.mdx:9]: corrected

File: docs/administration/appliance-manager/enable-or-disable-tls-versions-from-appliance-manager.md (hunk 1 above)
Comment (Nadia-JSch) [reply docs/administration/appliance-manager/enable-or-disable-tls-versions-from-appliance-manager.md]: Updated the sentence and added bold formatting to UI options mentioned on the page

File: docs/auto-discovery/sccm-discovery-net-tool.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ sidebar_position: 26
 ---
 
 :::note
-This tool will be deprecated in the near future. From v18.11.00 of Device42, SCCM integration is available natively from the Device42 Main Appliance, or using the WDS service. You can continue to use the tool for now, but we recommend that you use the native functionality.
```
Comment (zaneD42) [inline docs/auto-discovery/sccm-discovery-net-tool.md:7]: Just remove this page entirely. It's been replaced by in product capabilities.

Comment (Nadia-JSch) [reply docs/auto-discovery/sccm-discovery-net-tool.md:7]: removed

**Pull request merged**